frappe/utils/data.py

```diff
@@ -246,7 +246,8 @@ def format_datetime(datetime_string, format_string=None):
 
 def global_date_format(date):
 	"""returns date as 1 January 2012"""
-	formatted_date = getdate(date).strftime("%d %B %Y")
+	date = getdate(date)
+	formatted_date = babel.dates.format_date(date, locale=(frappe.local.lang or "").replace("-", "_"))
```

Comment:
Hi @netchampfaris ,

Thanks for your review !

I have made the requested change:
![image](https://cloud.githubusercontent.com/assets/4903591/26760946/41db4fbe-4925-11e7-8c1d-b0aa89710e38.png)


Let me know if you need any other change.

Have a nice day!